### PR TITLE
Copying file

### PR DESCRIPTION
**Pull request by alficles**
Added copying file to list of expected license files.

At some point, it would be nice to pull this information directly from the LICENSE file, but I didn't include a signifier on those lines like I did with the `@` for the patterns, so it will need a breaking change or more careful thought for implementation.

### CODE REVIEW

**Comment by zrhoffman** [comment]
Fixes #19

File: license.go [hunk 1]
```diff
@@ -237,7 +237,7 @@ forUnknownFiles:
 		if len(licenses) == 0 {
 			parts := strings.Split(name, `/`)
 			for i := len(parts) - 1; i > 0; i-- {
-				for _, licName := range []string{`LICENSE`, `LICENCE`, `LICENSE.md`, `LICENCE.md`, `LICENSE.txt`, `LICENCE.txt`} {
+				for _, licName := range []string{`LICENSE`, `LICENCE`, `LICENSE.md`, `LICENCE.md`, `LICENSE.txt`, `LICENCE.txt`, `COPYING`, `COPYING.md`, `COPYING.txt`} {
```
Comment (zrhoffman) [inline license.go:240]: As long as there needs to be a whitelist, can `LICENSE.MIT` also be included? [go-errors/errors](/go-errors/errors) uses it.

**Review by zrhoffman — CHANGES_REQUESTED** [review]

File: license.go (hunk 1 above)
Comment (alficles) [reply license.go:240]: That's a good observation, but I can't get all the possible variations, unfortunately. The LICENSE, LICENCE, and COPYING variations are all very standard places for license information that applies to a whole directory. You can use a .dependency_license file to explicitly document the license for anything that isn't automatically detectable.

**Review by zrhoffman — APPROVED** [review]
Tested new license names and they work as expected.

Documentation looks fine.

**Pull request merged**